tests/test_options.py

```diff
@@ -65,20 +67,21 @@ class MyEntity(Entity):
         except Exception:
             pass
 
-    def test_allowcoloverride_true(self):
-        class MyEntity(Entity):
-            name = Field(String(30))
-            using_options(allowcoloverride=True)
-
-        setup_all()
-
-        # Note that this test is bogus as you cannot just change a column this
-        # way since the mapper is already constructed at this point and will
-        # use the old column!!! This test is only meant as a way to check no
-        # exception is raised.
-        #TODO: provide a proper test (using autoloaded tables)
-        MyEntity._descriptor.add_column(Column('name', String(30),
-                                               default='test'))
+    # NOTE(je): allowcoloverride option does not appear to be implemented
+    # def test_allowcoloverride_true(self):
+    #
```

Comment:
`allowcoloverride` doesn't appear to be supported (this UT raises NotImplementedError) in the library and it isn't used in hermosa, so this UT was commented out.